machine_learning_hep/processerdhadrons_jet.py

```diff
@@ -355,24 +351,21 @@ def process_response(self):
             list_df_mc_reco.append(df_mc_reco)
 
         """ Here we can merge the dataframes corresponding to different HF pt in a
-        single one. In addition we are here selecting only non prompt HF"""
+        single one. In addition we are here selecting only non prompt HF """
 
         df_gen = pd.concat(list_df_mc_gen)
         df_mc_reco = pd.concat(list_df_mc_reco)
-        z_array_gen = z_calc(df_gen.pt_jet, df_gen.phi_jet, df_gen.eta_jet,
-                             df_gen.pt_cand, df_gen.phi_cand, df_gen.eta_cand)
-        df_gen["z"] = z_array_gen
-
-        zarray_reco = z_calc(df_mc_reco.pt_jet, df_mc_reco.phi_jet, df_mc_reco.eta_jet,
-                             df_mc_reco.pt_cand, df_mc_reco.phi_cand, df_mc_reco.eta_cand)
-        df_mc_reco['z'] = zarray_reco
 
-        zarray_gen = z_gen_calc(df_mc_reco.pt_gen_jet, df_mc_reco.phi_gen_jet,
-                                df_mc_reco.eta_gen_jet, df_mc_reco.pt_gen_cand,
-                                df_mc_reco.delta_phi_gen_jet, df_mc_reco.delta_eta_gen_jet)
+        """ add the z columns """
```

Comment:
See above and following comments. Instead of using """ use "#"

Reply:
Fixed